macos/FluentUI/Button/Button.swift

```diff
@@ -154,6 +154,39 @@ open class Button: NSButton {
 		}
 	}
 
+	/// When set to a non `nil` value ,the Secondary Image will always be placed on the trailing edge of the button in both LTR and
+	/// RTL language settings. It is designed to remain independent of  the `imagePosition` value of the NSButton which deals
+	/// with the main Button `image`.
+	public var secondaryImage: NSImage? {
```

Comment:
Minor: Maybe put this right below the "image" property to keep them together, since these aren't really sorted alphabetically anyhow.